src/signalcorr.jl

```diff
@@ -89,10 +103,17 @@ autocov{T<:Real}(x::VecOrMat{T}; demean::Bool=true) = autocov(x, default_autolag
 
 ## autocor
 
+"""
+    autocor!(r, x, lags; demean=true)
+
+Compute the autocorrelation function (ACF) of a vector or matrix `x` at `lags`
```

Comment:
OK. About the naming, I don't know how other software call this, but `autocov` sounds more explicit than `acf`, and it allows to the distinction with `autocor`. Though `pacf` clearly stands out; not sure what to do about it: maybe `pautocor`? Could file an issue to keep track of this.